app/Http/Controllers/LoginController.php

```diff
@@ -23,7 +23,7 @@ public function doLogin()
 
         if (Auth::attempt(['email' => $user->email, 'password' => $user->password])) {
             // Authentication passed...
-            return redirect('home')->with('status', 'You are logged in');
+            return redirect('admin/home')->with('status', 'You are logged in as administrator');
```

Comment:
untuk pemanggilan route lebih baik menggunakan nama route-nya instead of uri-nya cc @kharisisriyant @jovianch